app/models/work.rb

```diff
@@ -723,5 +723,11 @@ def publish_precurated_files
 
       delete_pre_curation_s3_dir
     end
+
+    def notify_collection_curators(current_user)
+      curators = collection.administrators.map { |admin| "@#{admin.uid}" }.join(", ")
+      notification = "#{curators} The [work](#{work_path(self)}) is ready for review."
```

Comment:
I'm curious whether this is a full url or just a path... but that's something for me to track down.

Reply:
Just a relative path. I see your point, a path might not be helpful in an email.